apps/auth_content/admin.py

```diff
@@ -0,0 +1,31 @@
+from django.contrib import admin
+
+from import_export.admin import ExportMixin
+
+from .models import ClosedPODContactInformation
+
+
+@admin.register(ClosedPODContactInformation)
+class ClosedPODContactInformationAdmin(ExportMixin, admin.ModelAdmin):
+    list_display = (
+        "id",
+        "user",
+        "facility_name",
+        "facility_id",
+        "primary_contact_name",
+        "secondary_contact_name",
+    )
+    list_filter = (
+        "user",
```

Comment:
I wonder if it would be a better user experience to not have any filters, since there will probably be a lot of submissions, each with its own `user`, `facility_name`, `facility_id`, `primary_contact_name`, and `secondary_contact_name`, which will:
- make this page take a long time to load
- make it difficult to find each of the filters

For example, if there are 100 users, I wouldn't want to scroll past all of them in order to find the facility name filter.

Reply:
Sounds good. Removed.